com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/UsageDataLogHandler.java

```diff
@@ -31,6 +32,10 @@ public void publish(LogRecord record) {
         if (record.getLevel().intValue() >= thresholdLevel.intValue()) {
             if (record.getThrown() != null) {
```

Comment:
Since `record.getThrown()` occurs 3 times around here, for me, I'd prefer assigning it to a local var.